whiteboarding/README.md

```diff
@@ -0,0 +1,16 @@
+### Platforms to use during whiteboarding
+
```

Comment:
Let's put a sentence here that says something along the lines of: "If you are currently in foundations, please go straight to the leetcode.md file above and solve the problems on either vs code or the leet code platform. For debugging purposes, we suggest using vs code then copying your code on to leet code.  You should not worry about the section below until you have gotten to the first official whiteboarding session which takes place during week 4."